src/pages/About/About.tsx

```diff
@@ -0,0 +1,149 @@
+import "./About.scss";
+import ballet from "../../assets/images/ballet.png";
+import reading from "../../assets/images/reading.png";
+import Header from "../../components/Header/Header";
+
+const About = () => {
+  return (
+    <div className="about">
+      <Header
```

Comment:
Can you combine this content with the <Layout/> component? At the moment you lose the navbar. Have a look at the Home component to see how it is implmented.